docs/BROWSERSETUP.md

```diff
@@ -47,9 +46,3 @@ When no browser entry is specified, that means all options and capabilities are
 #### Key "options"
 When a test should run with browser-specific option settings the key for "options" must be set.  
 For the browser-specific values please refer to the particular browser vendor.
-
```

Comment:
Would it make sense to note (just as a footnote) that capabilities existed in the past but were dropped in favor of options? If someone makes use of an older browser he might run into issues because capabilities are no longer supported ...